### ISSUE TITLE
Form 2.0 "Delete" does nothing- can't delete

### ISSUE THREAD

**Issue by suzialeksander** [geneontology/noctua-form#95]
In Noctua Form 2.0, I added a CC only annotation.  Tried to add another CC, but it added as an extension of the first instead of as an individual annotation, so I tried to use the three-dot "Add/Evidence/Delete" dropdown on the far right, the "delete" option at the bottom.  Nothing happens. Refreshed my model, unwanted CC still there.  

Model gomodel:5e7245050000237 if it matters but I'm deleting the unwanted one. 

Edit: I am not deleting the unwanted one as I cannot, I thought I could clear the blanks individually but I'm getting some (fun?) errors like the reference and the with are now both "[object Object] ".  I'm trying to delete the annotation with extension GO:0005886.

**Comment by vanaukenk** [comment]
@suzialeksander 
We need to remove the 'Delete' functionality from the form, as we weren't certain we could accurately indicate what the behavior would be in highly connected models, although this is something I'd like to re-visit.

**Comment by vanaukenk** [comment]
Also @suzialeksander 
When you said you wanted to add another CC annotation, were you wanting to add an entirely separate annotation?  If so, you'd need to go back to the Form to add it.  Once a CC annotation is made, any additional CC annotations will be considered annotation extensions.

**Comment by suzialeksander** [comment]
Yes, it was to the same GP but different evidence: IDA for plasma membrane, ISM to integral, I don't think we (SGD) have a SOP of cleanly combining those in multi-evidence Noctua annots yet so kept them separate for now. I didn't think an extension made sense.

I did add the extra annotation it as a new CC only annotation and I think it now works, thanks! I think the only issue out of this is the delete not working, which it seems you're aware of.

**Comment by vanaukenk** [comment]
Okay, thanks @suzialeksander 
I'll just leave this ticket open until we remove the delete option.

**Comment by vanaukenk** [comment]
See also #120 and #121

**Comment by krchristie** [comment]
I encountered this today and found that this existing ticket seems to be the exact same issue, so I'll put my example below.

Despite the oddity that the  "**Delete Activity**" option only deletes what is shown in the Form, and not the unshown extension for the anatomy term, it is still much faster to delete most of this annotation in the From and then finish the job by deleting the orphaned Anatomy term individual in the Graph, than it would be to have to individually delete each individual in the Graph. Until it is possible to do some sort of group delete easily in the Graph, I would strongly prefer that this "**Delete Activity**" option remain available, even with it's slightly incomplete behavior.

**My Example:**
In the form, I made an annotation that appeared to be an exact duplicate of another annotation (it actually wasn't due to the form not showing the effect of the "**Clone Activity**" allowing me to add a second CC term in the same annotation but not showing it in the form.) I used the "**Delete Activity**" to delete one of them. This is what the form and graph looked like before I used the "**Delete Activity**" option, noting that the Anatomy term shows in the Graph, but not in the Form:
![20220624-annotInForm](https://user-images.githubusercontent.com/10533218/175667098-d2abee7d-ee5d-4719-b1aa-8d93b9bffb39.png)
![20220624-annotInGraph](https://user-images.githubusercontent.com/10533218/175667110-41417858-273c-4902-8048-d4139b44a54d.png)

Here is what the Form looked like after I used the "**Delete Activity**" option, but BEFORE I refreshed the Form web page using shift reload:

![20220624-annotInForm-afterDel-without-Reload](https://user-images.githubusercontent.com/10533218/175667394-9206cc2c-cba0-4111-b758-5b2a0ba63efc.png)

Here is what the Graph looked like at the same time:
![20220624-annotInGraph-afterDelete](https://user-images.githubusercontent.com/10533218/175667465-4d453869-2243-4ad7-8148-2a4fac9193a4.png)

Here is what the Form looked like when I reloaded the Form web page using shift reload, but BEFORE I deleted the stranded Anatomy Term individual from the Graph. Now only the other two annotations (not shown in the other 2 screenshots of the Form) are present, and the one to "**basal pole of outer hair cell**" is gone:
![20220624-annotInForm-afterDel-n-Reload](https://user-images.githubusercontent.com/10533218/175667517-b373bbbd-5aeb-4305-9e24-9af39129a6f3.png)
